app/src/main/java/uk/gov/android/securestore/authentication/UserAuthenticator.kt

```diff
@@ -38,4 +42,8 @@ internal class UserAuthenticator(
             AccessControlLevel.PASSCODE_AND_CURRENT_BIOMETRICS ->
                 BIOMETRIC_STRONG or DEVICE_CREDENTIAL
         }
+
+    override fun close(context: Context) {
+        fragmentContext.finish()
```

Comment:
As discussed, make fragmentContext nullable at declaration and set it to null here

Reply:
Done